playbooks/roles/bats/defaults/main.yaml

```diff
@@ -0,0 +1,5 @@
+---
+bats_git_dir: "/root/bats"
+bats_forklift_dir: "/root/forklift"
+bats_update_forklift: "yes"
+bats_tests: "fb-content-katello.bats fb-capsule.bats fb-finish-katello.bats"
```

Comment:
Would be nice if we could use the "named" versions here the same as when you call `katello-bats` (e.g. `katello-bats content`)